.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ on:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
 jobs:
   release:
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
```

Comment:
Need to use ubuntu as windows can't build for linux, however linux can build for windows